src/WebJobs.Script.Grpc/Server/GrpcServer.cs

```diff
@@ -15,10 +15,10 @@ public class GrpcServer : IRpcServer, IDisposable
         private Server _server;
         private bool _disposed = false;
 
-        public GrpcServer(FunctionRpc.FunctionRpcBase serviceImpl, int grpcMaxMessageLength)
```

Comment:
TODO: Investigate if this setting used by any apps

Reply:
So for anyone currently using this setting in production, it will now be ignored? Technically a breaking change, but what does it mean for people practically? If they were previously guarded by this setting and getting errors for large messages, they'd now get through?

Reply:
We did run a few queries and did not find any usage of this.